x-pack/plugins/stack_connectors/public/connector_types/d3security/d3security.tsx

```diff
@@ -8,7 +8,7 @@
 import { lazy } from 'react';
 import { i18n } from '@kbn/i18n';
 import { GenericValidationResult } from '@kbn/triggers-actions-ui-plugin/public/types';
-import { SUB_ACTION } from '../../../common/gen_ai/constants';
+import { SUB_ACTION } from '../../../common/open_ai/constants';
```

Comment:
I wonder why the D3 connector uses `open_ai` sub-actions.. 🤔

Reply:
wow good catch